webview-ui/src/utilities/runtimeTypes.ts

```diff
@@ -0,0 +1,7 @@
+export function isObject(value: unknown): value is object {
```

Comment:
Note - this file is duplicated between the webview and extension projects. In the future it might be good to create a library of files that can be shared between the two (those that have no dependency on vscode or react). A discussion for later...